README.md

```diff
@@ -109,7 +109,7 @@ This project uses the `project.json` for the project build system. Please see th
 
 To build the Open XML SDK:
 - Clone the repo at https://github.com/OfficeDev/Open-XML-SDK
-- Open the solution using Visual Studio 2015.  Community Edition of VS2015 works just fine.
+- Open the solution using Visual Studio 2017.  Community Edition of VS2017 works just fine.
```

Comment:
VS 2017 is actually not the only supported tool. I'd rephrase along the lines of "Open the solution with an editor that supports the latest .NET project files, such as VS 2017, VS Code, VS for Mac, or .NET CLI. See https://www.microsoft.com/net/download/core for details."